app/src/main/java/com/folen/androidshowreel/activity/MainActivity.java

```diff
@@ -1,15 +1,56 @@
 package com.folen.androidshowreel.activity;
 
 import android.os.Bundle;
-
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import com.folen.androidshowreel.R;
 import com.folen.androidshowreel.base.BaseActivity;
+import com.folen.androidshowreel.model.Features;
+import com.mikepenz.fastadapter.FastAdapter;
+import com.mikepenz.fastadapter.adapters.ItemAdapter;
+
+import java.util.Arrays;
 
 public class MainActivity extends BaseActivity {
 
+    RecyclerView recyclerView;
+    private FastAdapter<Features> fastAdapter;
+    private ItemAdapter<Features> itemAdapter;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Bindings();
+        Setters();
+        AddItems();
+    }
+
+    public void Bindings() {
+        recyclerView = findViewById(R.id.recycler_view);
+        itemAdapter = new ItemAdapter<>();
+        fastAdapter = FastAdapter.with(Arrays.asList(itemAdapter));
+    }
+
+    public void Setters() {
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+        recyclerView.setHasFixedSize(true);
+        recyclerView.setItemAnimator(new DefaultItemAnimator());
+        recyclerView.getItemAnimator().setAddDuration(500);
+        recyclerView.getItemAnimator().setRemoveDuration(500);
+        recyclerView.setAdapter(fastAdapter);
+    }
+
+    public void AddItems() {
+        itemAdapter.add(new Features(getString(R.string.list_data_pass),getString(R.string.list_data_pass_description),R.mipmap.ic_launcher));
```

Comment:
Zrob sobie plik JSON gdzie bedziesz przechowywal sobie te swoje dane

cos takiego tylko uzupelnij to prawdziwymi  danymi, rozne id, rozne tytuły i linki do jakichs obrazkow z neta:

```
{
  "operations": [
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "9eb259ba-1842-4370-a2fc-fd7531f95a76"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "c765745a-8844-4c94-af01-4e34339ffc6a"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "1c3e1000-f5e5-436c-9b1d-be0a641befa5"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "0a8d35da-6e66-4ed7-8ce5-efff27f9701c"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "2e646762-c71d-46fc-8590-b0f9d0db0ecf"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "829726c9-ae66-4b19-98eb-8af3c5fb7169"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "dc73f056-3038-47b4-8aa7-9211805ef91b"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "68b8e0ed-b2a4-4c7f-893a-4198daa6d878"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "4d513a2b-9ee3-46a2-accc-1b06a98f5024"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "56701c1d-f77b-4793-81a9-8622fa34918f"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "f8408a13-fb98-4026-b674-768e933fd76a"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "f87bce0d-5058-459e-a9b6-2449558c22fb"
    },
    {
      "id": 0,
      "title": "s",
      "description": "s",
      "img": "0f1b79df-c8e8-459f-9de7-eecc4a84f4ab"
    }
  ]
}
```

Bedziesz musial do tego sobie dopisac jakas funkcje ktora bedzie miala switcha na tego id'ka i w zaleznosci od jakiegos id'ka bedziesz musial zwracac Intent. To ci posluzy do tego zebys na podstawie danych z JSONa wiedzial co masz otworzyc po nacisnieciu na element listy